package.json

```diff
@@ -80,12 +80,15 @@
     "karma": "^0.13.22",
     "karma-browserify": "^5.0.3",
     "karma-jasmine": "^0.3.8",
-    "karma-phantomjs-launcher": "^1.0.0",
+    "karma-phantomjs-launcher": "^1.0.4",
     "karma-requirejs": "1.1.0",
-    "phantomjs-prebuilt": "^2.1.7",
+    "phantomjs-prebuilt": "^2.1.15",
     "requirejs": "2.1.0",
     "sinon": "2.3.8",
     "watchify": "^3.7.0",
     "webpack": "^1.13.3"
+  },
+  "resolutions": {
+    "reveal.js": "3.5.0"
```

Comment:
Why are we coupling Asciidoctor.js core to reveal.js? This seems like it doesn't belong.

Reply:
I'm using `asciidoctor-reveal.js` in the tests suite to check that the integration between Asciidoctor.js and Asciidoctor Reveal.js is working.

Asciidoctor.js is used in a variety of environments and I want to make sure that I don't unintentionally break compatibility. I see it as a "smoke test" but arguably we could have a dedicated project to do that :nerd_face: 

Anyway, Asciidoctor Reveal.js is using Reveal.js 3.3.0 and this version does not allow to use a recent version of Node: `"node": "~4.1.1"` (see https://github.com/hakimel/reveal.js/blob/539e774d31f91676bcc3f75e28168921cd27d819/package.json#L22).
As opposed to `npm`, `yarn` strictly enforces this requirement, so in order to use `yarn`, I had to force the version. In fact, version 3.5.0 now allows to use any version of Node >= 4: `"node": ">=4.0.0"`.